src/cascadia/TerminalControl/TermControl.h

```diff
@@ -103,6 +103,7 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         TermControl(IControlSettings settings, TerminalConnection::ITerminalConnection connection);
 
         winrt::fire_and_forget UpdateSettings();
+        winrt::fire_and_forget UpdateAppearance(IControlAppearance newAppearance);

```

Comment:
same comment as earlier about the control just getting the appearance from the `IControlSettings`. If the control does that, then it probably doesn't need a separate `UpdateAppearance` method publicly exposed

Reply:
decided we are keeping this from teams discussion